### DISCUSSION TITLE
Welcome to arduino-web-timers Discussions!

### DISCUSSION THREAD

**Discussion by dbuezas** [dbuezas/arduino-web-timers#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kamilsss655** [comment]
Thank you so much for your work! This has been extremely useful tool for my LTG8 project! :+1:

**Comment by rushwind** [comment]
This is pretty amazing and I am sure a lot of work. Thank you!!!!
